homeassistant/components/hassio/repairs.py

```diff
@@ -124,7 +131,8 @@ async def _async_step_apply_suggestion(
         if not confirmed and suggestion.key in SUGGESTION_CONFIRMATION_REQUIRED:
             return self._async_form_for_suggestion(suggestion)
 
-        if await async_apply_suggestion(self.hass, suggestion.uuid):
+        with suppress(SupervisorError):
```

Comment:
Use try... except instead and put the abort under the except case and create entry outside the try... except. That way we only wrap the line that can raise in the try... except block.